File: happykopiAPI/happykopiAPI/DTOs/Transaction/Outgoing_Data/TransactionSummaryDto.cs
namespace happykopiAPI.DTOs.Transaction.Outgoing_Data
{
    public class TransactionSummaryDto
    {
        public decimal TotalSalesToday { get; set; }
        public int TransactionsToday { get; set; }
        public int CashPayments { get; set; }
        public int CashlessPayments { get; set; }
    }
}

File: happykopiApp/src/app/core/dtos/transaction/transaction-summary.dto.ts
import { TransactionItemDto } from "./transaction-detail.dto";
import { TransactionListItemDto } from "./transaction-list-item.dto";

export interface TransactionSummaryDto {
  totalSalesToday: number;
  transactionsToday: number;
  cashPayments: TransactionListItemDto;
  cashlessPayments: TransactionListItemDto;
}